service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/CaseInitiationControllerTest.java

```diff
@@ -86,4 +100,31 @@ void shouldPopulateErrorsInResponseWhenDomainNameIsNotFound() throws Exception {
 
         assertThat(response.getResponse().getContentAsString()).isEqualTo(MAPPER.writeValueAsString(expectedResponse));
     }
+
+    @Test
+    void submittedEndpointShouldReturnOkResponse() throws Exception {
+        given(serviceAuthorisationApi.serviceToken(anyMap()))
+            .willReturn(SERVICE_AUTH_TOKEN);
+
+        CallbackRequest request = CallbackRequest.builder().caseDetails(CaseDetails.builder()
+            .id(1L)
+            .data(ImmutableMap.<String, Object>builder()
+                .put("caseLocalAuthority", "example")
+                .build()).build())
+            .build();
+
+        mockMvc
+            .perform(post("/callback/case-initiation/submitted")
+                .header("authorization", AUTH_TOKEN)
+                .header("user-id", USER_ID)
+                .contentType(MediaType.APPLICATION_JSON)
+                .content(MAPPER.writeValueAsString(request)))
+            .andExpect(status().isOk());
+
+        Thread.sleep(3000);
+
+        verify(caseAccessApi, times(3)).grantAccessToCase(
```

Comment:
Please make sure correct values are passed to `grantAccessToCase` method. We should be as strict as we only can.